### PR TITLE
add zbar library

### PR DESCRIPTION
**Pull request by DadaTrumpet**

### CODE REVIEW

**Comment by JPeterMugaas** [comment]
When I built it, I got errors about invalid file formats.  But I typically build without cleaning all of the sources.

  [[ -d "${srcdir}"/build-${CARCH} ]] && rm -rf "${srcdir}"/build-${CARCH}
  mkdir -p "${srcdir}"/build-${CARCH} && cd "${srcdir}"/build-${CARCH}
  ../${_realname}-${pkgver}/configure \

then you would make from the "${srcdir}"/build-${CARCH} directory.  That way, you can have two sets of compiled binaries that are separate from eachother.

**Comment by Alexpux** [comment]
Also reset pkgrel to 1

**Comment by mati865** [comment]
And fix maintainer/contributor fields

**Comment by Alexpux** [comment]
@DadaTrumpet ping.

**Comment by DadaTrumpet** [comment]
Sorry for the delay, ok I look at it this afternoon.


<http://www.delair-tech.com/en/home/>
   <http://fr.linkedin.com/company/delair-tech>
<https://www.youtube.com/channel/UCx8pnSVCQqHoL7clawE5VcA>
<https://www.instagram.com/delairtech/>
<https://twitter.com/DelairTech?lang=en>
<https://www.facebook.com/DelairTech/>


*David SEGUELA*

*Software engineer - Image processing*


*676 rue Max Planck 3670 Labèg**e*

Tel: +33 971169937 <%2B33%209%2072%2050%2056%2034>

www.delair-tech.com




2017-06-04 11:57 GMT+02:00 Алексей <notifications@github.com>:

File: mingw-w64-zbar/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.10
+pkgrel=1
+pkgdesc="Application and library for reading bar codes from various sources"
+arch=('any')
+url="http://zbar.sourceforge.net/"
+license=('LGPL')
+depends=("${MINGW_PACKAGE_PREFIX}-imagemagick")
+source=("http://downloads.sourceforge.net/project/${_realname}/${_realname}/${pkgver}/${_realname}-${pkgver}.tar.bz2"
+		"windowsandimagemagick.patch")
```
Comment (Alexpux) [inline mingw-w64-zbar/PKGBUILD]: Use spaces everywhere instead tabs

**Pull request merged**